Library/Output Templates/fantasy_grounds_npc_template.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+@ENHANCED_KEY_PARSING
+@ENCODING_OFF
+<root version="2.8">
+  <npc>
+    <name type="string">@NAME</name>
+    <notes type="formattedtext">
+      <p>Race: @RACE</p>
+      <p>height: @HEIGHT</p>
+      <p>weight: @WEIGHT</p>
+      <p>age: @AGE</p>
+      <p>@GENDER, Eyes: @EYES, Hair: @HAIR, Skin: @SKIN</p>
+      @NOTES_LOOP_START@@NOTE_FORMATTED@@NOTES_LOOP_END
+    </notes>
+    <pts type="number">@TOTAL_POINTS</pts>
+    <traits>
+      <sizemodifier type="string">@SIZE</sizemodifier>
+      <reactionmodifiers type="string"></reactionmodifiers>
+      <description type="string">
+        @ADVANTAGES_LOOP_START
+        @DESCRIPTION_PRIMARY (@POINTS)\r@ADVANTAGES_LOOP_END
```

Comment:
What is `\r` supposed to be doing?

Reply:
Fantasy Grounds converts "\r" to a newline when displaying the advantages.   The NPC format does not explicitly identify each advantage, but instead displays them in a semi-formatted text field.